radio/src/CMakeLists.txt

```diff
@@ -443,7 +443,7 @@ set(CMAKE_EXE_LINKER_FLAGS "")
 set(CMAKE_C_LINK_EXECUTABLE "<CMAKE_C_COMPILER> <FLAGS> <LINK_FLAGS> <OBJECTS>  -o <TARGET>")
 set(CMAKE_CXX_LINK_EXECUTABLE "<CMAKE_CXX_COMPILER> <FLAGS> <LINK_FLAGS> <OBJECTS>  -o <TARGET>")
 
-use_cxx11()  # ensure gnu++11 in CXX_FLAGS with CMake < 3.1
+use_cxx14()  # ensure c++14 in CXX_FLAGS with CMake < 3.1
```

Comment:
So which compiler versions does this knock out?

Reply:
Here's an answer for gcc: https://stackoverflow.com/questions/31965413/compile-c14-code-with-g
I wonder if CMAKE knows which flag to use for older gcc with `CMAKE_CXX_STANDARD 14` (but the `use_cxx14()` macro is a bit simplistic).